staging/src/k8s.io/apimachinery/pkg/util/proxy/transport.go

```diff
@@ -130,12 +134,7 @@ func (rt *Transport) WrappedRoundTripper() http.RoundTripper {
 // to the same host as sourceURL, which is the page on which the target URL
 // occurred, or if the URL matches the sourceRequestHost. If any error occurs (e.g.
 // parsing), it returns targetURL.
```

Comment:
Can you remove the last sentence of the comment, which is not relevant anymore?

Reply:
good catch :+1: